logic_mod/Blocks/Api/CpuQuat.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Logic.Script;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace Logic.Blocks.Api
+{
+    public class CpuQuat : ApiList
+    {
+        public override List<CpuApiFunc> Api => new List<CpuApiFunc>
+        {
+            new CpuApiFunc("mul", false, "multiply quaternion",
+                new Dictionary<string, CpuApiFunc.ArgInfo>{
+                    { "a", new CpuApiFunc.ArgInfo("quaternion", "quaternion to multiply") },
+                    { "b", new CpuApiFunc.ArgInfo("quaternion or vector", "quaternion or vector to multiply") }
```

Comment:
The ArgInfos are used to construct api description. Such info will lead to generating the following help in "cpuapi" command:
  Quaternion.mul(quaternion a, quaternion or vector b)
Having spaces in argument type seems wrong.